disperser/batcher/batcher.go

```diff
@@ -426,6 +427,7 @@ func (b *Batcher) HandleSingleBatch(ctx context.Context) error {
 	for quorumID := range batch.State.Operators {
 		quorumIDs = append(quorumIDs, quorumID)
 	}
+	slices.Sort(quorumIDs)
```

Comment:
Out of curiosity, what happens if they aren't sorted?

Reply:
[QuorumApks](https://github.com/Layr-Labs/eigenda/blob/master/core/eth/tx.go#L492) can be out of order and [signature validation](https://github.com/Layr-Labs/eigenlayer-middleware/blob/dev/src/BLSSignatureChecker.sol#L207) can fail